frontend/common/src/components/SkillPicker/SkillPicker.tsx

```diff
@@ -241,7 +244,10 @@ const SkillPicker = ({
           </Chips>
         </>
       ) : null}
-    </>
+      {submitButton && handleSave && (
+        <form onSubmit={handleSubmit(handleSave)}>{submitButton}</form>
```

Comment:
Is it required to use a `<form>` element here? Could not just remove the form and have the button passed in call `onSave` instea?

Reply:
So ya it is required for the Submit button text to work, since it uses the [`useForm()`](https://react-hook-form.com/api/useform/handlesubmit/) hook. I've tried some other ways including the way you suggested, but it didn't work. I'm still reading through the docs for a better solution.

Reply:
Turns out I just needed to use onClick instead of onSubmit 😆